cmake/FindPoly2Tri.cmake

```diff
@@ -0,0 +1,23 @@
+# Find Poly2Tri
+# ~~~~~~~~~
+# Copyright (c) 2020, Peter Petrik <zilolv at gmail dot com>
+# Redistribution and use is allowed according to the terms of the BSD license.
+# For details see the accompanying COPYING-CMAKE-SCRIPTS file.
+#
+#
+# Once run this will define: 
+#  Poly2Tri_FOUND - System has Poly2Tri
+#  Poly2Tri_INCLUDE_DIR - The Poly2Tri include directory
+#  Poly2Tri_LIBRARY - The library needed to use Poly2Tri
+
+FIND_PATH(Poly2Tri_INCLUDE_DIR poly2tri.h
```

Comment:
@3nids recently modernised all our cmake to change commands to lowercase and remove the () contents from else/endif statements. Can you do that here too for consistency?

Reply:
if all modernisation would be that easy..